cookbooks/swift/recipes/tracing_info.rb

```diff
@@ -0,0 +1,48 @@
+#
+#Copyright (c) 2015-2022, NVIDIA CORPORATION.
+#SPDX-License-Identifier: Apache-2.0
+
+execute "Start jaeger all-in-one docker image" do
+  command "/vagrant/bin/reset_jaeger.sh"
+end
+
+log 'show jaeger docker info' do
+  message %(
+  A Jaeger all-in-one has been started in the vagrant environment. It was started with the bin/reset_jaeger tool.
+  You can view all your traces at: http://saio:16686/search
+
+  The reset_jaeger.sh script basically runs:
+
+    docker run -d --name jaeger \\
+    -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \\
+    -p 5775:5775/udp \\
+    -p 6831:6831/udp \\
+    -p 6832:6832/udp \\
+    -p 5778:5778 \\
+    -p 16686:16686 \\
+    -p 14268:14268 \\
+    -p 14250:14250 \\
+    -p 9411:9411 \\
+    jaegertracing/all-in-one:1.27
+
+  See: https://www.jaegertracing.io/docs/1.27/getting-started/
+
+  If you want to reset and clear the traces just run:
+
+    reset_jaeger.sh
+
+  NOTE: We should go via OTel collector, but I havn't implemented that yet. But there are some notes on that below
+
+  For the OTel collector we should be able to run something like:
+
+    docker pull otel/opentelemetry-collector:latest
+    docker run -d otel/opentelemetry-collector:latest
+
+  See: https://opentelemetry.io/docs/collector/getting-started/
+  NOTE: of course you can also specify a version. We should probably pick whatever we use for prod (when we get that far)
+
+  If you want to have a custom config, volume mount in a config:
+
+    docker run -v $(pwd)/config.yaml:/etc/otelcol/config.yaml otel/opentelemetry-collector
```

Comment:
Please get rid of all this extra output at the end of `vagrant up`

either put it in a `bin/` script or docs

Reply:
@matthewoliver if we can get rid of this extra warnings in the end after provisioning the vagrant VM, you have my approval as well to get this merged. The tool has been helpful in tracing paths for each requests made and latency decomposition within each function invoked while interacting with swift using the swift client